.github/ISSUE_TEMPLATE/mod_integration.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Mod integration 
+about: Suggest an integration for a mod
+labels: enhancement, "mod-integration"
```

Comment:
The second label, 'mod-integration', doesn't exist yet, but it seems appropriate.

Reply:
I've created an `integration` label.